storage/local/crashrecovery.go

```diff
@@ -395,7 +395,7 @@ func (p *persistence) cleanUpArchiveIndexes(
 		if _, err := p.archivedFingerprintToMetrics.Delete(fp); err != nil {
 			return err
 		}
-		series := newMemorySeries(clientmodel.Metric(m), false, clientmodel.Earliest)
+		series := newMemorySeries(clientmodel.Metric(m), false, clientmodel.Earliest, clientmodel.Earliest)
```

Comment:
I added `clientmodel.Earliest` here to make it compile for now - haven't checked yet what it really should be in this context.